bootstrapper/inputs/inputsNg2Bootstrapper.ts

```diff
@@ -64,18 +64,12 @@ export class InputsBootstrapper {
 		this.selections = [this.typeaheadOptions[0], this.typeaheadOptions[2]]
 	}
 
-	wait(data: any): Observable<any> {
-		const subject: BehaviorSubject<ITestItem[]> = new BehaviorSubject<ITestItem[]>([]);
-		setTimeout(() => subject.next(data), 1000);
-		return subject.asObservable();
-	}
-
 	getOptions = (): Observable<any> => {
 		return Observable.of(this.typeaheadOptions);
 	}
 
 	searchOptions = (search: string): Observable<any> => {
-		return this.wait(filter(this.typeaheadOptions, option => option.value.search(search) != -1));
+		return Observable.of(filter(this.typeaheadOptions, option => option.value.search(search) != -1)).delay(1000);
```

Comment:
Could've applied this filter in a map() I guess, but that might be overkill, since this is all in the bootstrapper anyway.